distributed/asyncio.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
Which one do You prefer? It's totally up to You. Currently I've removed the top level import.